src/backend/joanie/settings.py

```diff
@@ -199,6 +200,7 @@ class Base(Configuration):
         "joanie.badges",
         "joanie.demo",
         "joanie.signature",
+        "drf_spectacular",
```

Comment:
do we really want to add it in every environment ?

Reply:
I added it here because of https://app.circleci.com/pipelines/github/openfun/joanie/3894/workflows/109878cb-17ce-4a19-ae1b-8ccee3eadc21/jobs/41126